cmd/connector/customizer_init.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2023, SailPoint Technologies, Inc. All rights reserved.
+package connector
+
+import (
+	"embed"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+//go:embed static/customizer/*
+var customizerStaticDir embed.FS
+
+const (
+	customizerDirName      = "customizer"
+	customizerTemplatePath = "static/" + customizerDirName
+)
+
+func newCustomizerInitCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "init <customizer-name>",
+		Short:   "Initialize new connector customizer project",
+		Long:    `init sets up a new TypeScript project with sample connector customizer included for reference.`,
+		Example: "sail conn customizers init \"My Customizer\"",
+		Args:    cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			projName := args[0]
+			if projName == "" {
+				printError(cmd.ErrOrStderr(), errors.New("connector customizer name cannot be empty"))
+				return
+			}
+
+			if f, err := os.Stat(projName); err == nil && f.IsDir() && f.Name() == projName {
+				printError(cmd.ErrOrStderr(), fmt.Errorf("Error: project '%s' already exists.\n", projName))
+				return
+			}
+
+			if err := createDir(projName); err != nil {
+				_ = os.RemoveAll(projName)
+				printError(cmd.ErrOrStderr(), err)
+				return
+			}
+
+			err := fs.WalkDir(customizerStaticDir, customizerTemplatePath, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				}
+
+				if d.Name() == customizerDirName {
+					return nil
+				}
+
+				if d.IsDir() {
+					if err := createDir(filepath.Join(projName, d.Name())); err != nil {
+						return err
+					}
+				} else {
```

Comment:
nit, need this as else? of we cn do if{return}

Reply:
Do need this else to avoid copying directory. The else case will be reading and creating, copying file